qf_lib/tests/unit_tests/backtesting/portfolio/dummy_ticker.py

```diff
@@ -31,3 +33,15 @@ def from_string(cls, ticker_str: Union[str, Sequence[str]]) \
             return DummyTicker(ticker_str)
         else:
             return [DummyTicker(t) for t in ticker_str]
+
+
+class DummyExchangeTicker(CurrencyExchangeTicker):
```

Comment:
If we will create only a single CurrencyExchangeTicker for all DataProviders (which should be sufficient for our purposes), then we can remove this class and use directly the CurrencyExchangeTicker in the tests